pymedphys/tests/experimental/pseudonymisation/test_pseudonymisation.py

```diff
@@ -77,29 +83,31 @@ def test_identifier_with_unknown_vr():
 
 
 @pytest.mark.pydicom
-def _test_identifier_is_sequence_vr():
+def test_identifier_is_sequence_vr():
     replacement_strategy = pseudonymisation_api.pseudonymisation_dispatch
     logging.info("Using pseudonymisation strategy")
     identifying_keywords_no_SQ = ["PatientID", "RequestedProcedureID"]
-    identifying_keywords_with_SQ_vr = identifying_keywords_no_SQ.append(
-        "RequestAttributesSequence"
-    )
-    logging.info("Using keyword with VR = UR")
+    identifying_keywords_with_SQ_vr = [
+        "PatientID",
+        "RequestedProcedureID",
+        "RequestAttributesSequence",
+    ]
 
-    ds_input = pydicom.Dataset()
-    ds_input.PatientID = "ABC123"
-    request_attributes_seq = dicom_dataset_from_dict(
+    ds_input = dicom_dataset_from_dict(
         {
+            "PatientID": "ABC123",
             "RequestAttributesSequence": [
                 {
                     "RequestedProcedureID": "Tumour Identification",
```

Comment:
Just a preference nit-pick thing here. Can we define run:

```python
identifying_string = "Tumour Identification"
```

And then use that here:

```python
"RequestedProcedureID": identifying_string,
```

And then below change the following:

```python
    assert (
        ds_anon.RequestAttributesSequence[0].RequestedProcedureID
        != "Tumour Identification"
    )
```

to:

```python
    assert (
        ds_anon.RequestAttributesSequence[0].RequestedProcedureID
        != identifying_string
    )
```

Reason being, having a test potentially pass one day due to a typo raises a small alarm bell.